summon-citation_export.gemspec

```diff
@@ -19,4 +19,5 @@ Gem::Specification.new do |gem|
 
   gem.add_development_dependency 'rake'
   gem.add_development_dependency 'rspec'
+  gem.add_development_dependency 'byebug'
```

Comment:
I'm not sure if this is something we want to do or not. Maybe have a discussion about it? Is this normal?